CMakeLists.txt

```diff
@@ -144,6 +187,9 @@ if(CMAKE_COMPILER_IS_GNUCXX OR "${CMAKE_CXX_COMPILER_ID}" MATCHES "^(Apple)?Clan
   if("${OPT_FLAGS}" STREQUAL "")
     set(OPT_FLAGS "-ggdb3 -O2 -march=native -mtune=native")
   endif()
+else()
+  message(WARNING "This project is assumed to be compiled with gcc or clang."
+  "No guarantees can be made if another compiler is used.")
```

Comment:
Space in the string?

Reply:
Should be fixed in https://github.com/clearmatics/zeth/pull/208/commits/2be3993bf0895c2afc8b957158c31b94f61ff2e6